src/promptflow/promptflow/_sdk/operations/_local_azure_connection_operations.py

```diff
@@ -9,6 +9,7 @@
 from promptflow._sdk._utils import interactive_credential_disabled, is_from_cli, is_github_codespaces, print_red_error
 from promptflow._sdk.entities._connection import _Connection
 from promptflow._utils.logger_utils import get_cli_sdk_logger
+from promptflow.azure._utils.gerneral import get_base_token
```

Comment:
add a changelog for this change

Reply:
Done, but does the add category feature not need to be added to the changelog?